N3.js/src/main/scala/org/w3/banana/n3js/io/N3jsTurtleParser.scala

```diff
@@ -0,0 +1,45 @@
+package org.w3.banana
+package n3js
```

Comment:
Sorry, I expressed my point badly. I really meant to say that it differs to the other implementations, say in sesame. That said, I believe it is _better_ than what is there , so perhaps they could be updated too? But not a *must*